### PR TITLE
Added simplification to index.php and rudimentory personal page after login #5 and #1

### PR DESCRIPTION
**Pull request by MikelShifrin**
-created a new php folder with index folder inside where all previous parts of index.php are in seperate php files
-created a simple personal page after successful login
-only account that currently works is:
username: michael@gmail.com
password: Mike
problem existed before this pull request and was not yet addressed
changes can be viewed in action on: https://users.encs.concordia.ca/~m_shifr/

### CODE REVIEW

File: code/php/index/coursePage.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	<div class="dropdown" >
+				  <button class="btn">Courses</button>
+				  <div class="content" style="left:0;">
+						<a href="#">SOEN 341</a>
+						<a href="#">SOEN 490</a>
+						<a href="#">SOEN 287</a>
+						<a href="#">COEN 390</a>
+				  </div>
+				</div>			
+			</div>
```
Comment (Davidster) [inline code/php/index/coursePage.php]: Fix indentation in this file

File: code/php/index/loginForm.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<h1>Login Page</h1>
+<form id='login' action='' method='post'>
+<p>Enter your email:</p>
+<input type='email' name='email' placeholder='Email Address' required/>
+</br>
+<p>Enter your password:</p>
+<input type='password' name='password' placeholder='Password' required/>
+<input type='submit' value='Enter' name='logbtn'/>
+</form>	
+</div>";
```
Comment (Davidster) [inline code/php/index/loginForm.php]: Fix indentation in this file and remove `";` from the end of the file

File: code/php/index/registerPage.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+		<p>Confirm Password:</p>
+		<input type='password' name='repassword' placeholder='Password' required/>
+		<input type='radio' name='accountType' />
+		Teacher's Assistant
+		</br>
+		<input type='radio' name='accountType' />
+		Student
+		</br></br>	
+		<input type='submit' value = 'Submit' name='register'/>
+		</form>
```
Comment (Davidster) [inline code/php/index/registerPage.php]: Fix indentation on this line (21)

**Review by Davidster — APPROVED** [review]
Looks good - just fix the slight styling issues please

**Review by Megatron4537 — APPROVED** [review]
I like it!

**Comment by MikelShifrin** [comment]
There will be indentation problems from now on cause I start always to get the warning LS will be converted into CRLS. It's a Mac, Linux vs Windows spacing conversion problem. You will start having them aswell

**Comment by MikelShifrin** [comment]
I fixed all requested indentations and deleted the extra semicolon

**Pull request merged**